### ISSUE TITLE
Correctly describe the serialization format in the JSON schema

### ISSUE THREAD

**Issue by multimeric** [caniko/pydantic-numpy#37]
Currently, an `Np1DArrayFp64` will generate a JSON schema with the following type:
```
"type": "np.ndarray[tuple[int], np.dtype[float64]"
```

This is incorrect, as I don't think it's a valid JSON schema type at all. What it should return is the JSON schema describing a `NumpyDataDict`, which is already in the repo:
https://github.com/caniko/pydantic-numpy/blob/438b18c437ac0e4d34205d1f04d71b5e0bd119fa/pydantic_numpy/helper/typing.py#L7-L9

**Comment by multimeric** [comment]
Here's a reproducible example:

```python
>>> from pydantic.type_adapter import TypeAdapter
>>> from pydantic_numpy.typing import Np1DArrayFp64
>>> TypeAdapter(Np1DArrayFp64).json_schema()
{'strict_data_typing': False, 'type': 'np.ndarray[tuple[int], np.dtype[float64]'}
```

**Comment by multimeric** [comment]
Are you open to a PR fixing this? Because it's moderately important to my use case.

**Comment by caniko** [comment]
I will fix it, but it will have to wait till April. Welcome to make a PR if you can't wait in the meantime.

**Comment by caniko** [comment]
Fixed with #41
